cudapoa/src/cudapoa_kernels.cuh

```diff
@@ -242,24 +272,52 @@ __global__ void generatePOAKernel(uint8_t* consensus_d,
         // Run Needleman-Wunsch alignment between graph and new sequence.
         SizeT alignment_length;
 
-        if (cuda_banded_alignment)
+        if (banded_alignment || cuda_adaptive_banded)
```

Comment:
Sorry I missed this permutation! Can you please add a commit dropping the cuda_ prefix here?
Thanks!

Reply:
sure